ServerTaskHelper/AzureFunctionBasicHandler/README.md

```diff
@@ -0,0 +1,41 @@
+﻿### In basic example, Azure Function checks that the that the invoking pipeline contains at least one CmdLine task in the previous stages prior to the check point.
+
+This sample demonstrates the proper integration with Azure DevOps async checks, when the check condition takes a while to be evaluated due to an external dependency.
+Async check refers to a check which Completion event setting is configured to Callback.
+
+### The Azure Function goes through the following steps:
+
+1. Confirms the receipt of the check payload
+2. Sends a status update to Azure Pipelines that the check started
+3. Uses {AuthToken} to make a callback into Azure Pipelines to retrieve the pipeline run's Timeline entry
+4. Checks if the Timeline contains a CmdLine task
+5. Sends a status update with the result of the search
+6. Sends a check decision to Azure Pipelines
+
+### In order for this example to work, following instructions should be followed for the setup:
+1. Deploy azure function AzureFunctionBasicHandler to azure.
```

Comment:
azure function -> Azure Function? Maybe check other places in docs so we use consistent casing.